text.mustache.md

```diff
@@ -16,15 +16,15 @@ These terms add to the terms of the license for this software.  These terms beco
 {{#license}}
 ## Purpose
 
-This license gives everyone as much permission to work with this software as possible, while protecting contributors from liability and requiring that they be credited.
+I, the creator of this software, license to you, the user, as much permission to work with this software as possible, while protecting myself and contributors from liability and requiring credit for our work.
```

Comment:
So "I, the creator" is clearly singular. With multiple involved, it gets close to [possessory credit](https://en.wikipedia.org/wiki/Possessory_credit), which is drama-prone.

Reply:
hmm. you're right, that might further encourage the "BDFL" practice. Do you think we should try to soften distinction between I/we, or simply eliminate it?

Reply:
> Do you think we should try to soften distinction between I/we, or simply eliminate it?

I don't know! I will have to make some time to sit down and try my own hand at rewriting.

Pronoun number-agreement issues have dogged other licenses I've contributed to.  It's really hard to clarity and readability at the same time. Our language is very specific about one-versus-many, and very not specific about how many there are when there's more than one.